wordpress-mobile/WordPress-iOS
# Fix login with site address being blocked after failing the first attempt

Fixes #20985

Technical details explained in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/792

## To test:

1. On Login screen, tap Enter your existing site address (Alternatively, try to add a self-hosted site from the Site Picker)
2. Input a non-existing WordPress site address like `zzzz.zzz.zz`
3. Tap Continue and notice the error message
4. Type the existing site address without a scheme like `round-cheetah.jurassic.ninja`
5. There should be no error and username & password fields should be shown

### Before the fix

https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/7a38ae99-6deb-4c74-abc0-e73853da15ce

### After the fix

https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/1e698ad4-200d-432f-b98c-598ebe032e65


## Regression Notes
1. Potential unintended areas of impact

The fix is very precise and doesn't affect any other part of the authenticator. I cannot identify other areas that can be affected

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


UI Changes testing checklist:
